components/philosophical-dialogue.tsx

```
@@ -431,7 +456,14 @@ export function PhilosophicalDiscussion({ gameState, updateGameState }: Philosop
         </CardHeader>
         <CardContent className="space-y-4">
           <div className="bg-background p-4 rounded-lg border-l-4 border-purple-500">
-            <p className="italic leading-relaxed text-sm">{currentDialogue.content}</p>
+            {DIALOGUE_FORMATTING_ENABLED ? (
+              <div
+                className="italic leading-relaxed text-sm"
+                dangerouslySetInnerHTML={formatDialogueContent(currentDialogue.content)}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)